runtime/autoload/health/provider.vim

```diff
@@ -70,7 +70,8 @@ function! s:system(cmd, ...) abort
   let jobid = jobstart(a:cmd, opts)
 
   if jobid < 1
-    call health#report_error(printf('Command error (job=%d): %s', jobid, s:shellify(a:cmd)))
+    call health#report_error(printf('Command error (job=%d): `%s` (in %s)',
+          \ jobid, s:shellify(a:cmd), string(getcwd())))
```

Comment:
`getcwd()` already returns a string, so the coercion shouldn't be necessary.

Reply:
It's meant to put it in quotes - but I'm happy to remove that.

Reply:
Ah, ok.  Then I'm fine either way.